src/app/domain/geolocation/tracking/index.js

```diff
@@ -292,5 +293,5 @@ export const startOpenPathUploadAndPipeline = async ({
     Log('Failed openpath processing: ' + err)
     log.error(err)
   }
-  return
+  return uploadedCount
```

Comment:
you should update the jsdoc to reflect that change

Reply:
Right, it's done